lib/chorde/clients/__init__.py

```diff
@@ -1,18 +1,6 @@
 # -*- coding: utf-8 -*-
 import sys
 
-try:
-    from .. import lrucache
-except ImportError:
-    from .. import pylrucache as lrucache
-    sys.modules['chorde.lrucache'] = lrucache
-
-try:
-    from .. import cuckoocache
-except ImportError:
-    from .. import pycuckoocache as cuckoocache
-    sys.modules['chorde.cuckoocache'] = cuckoocache
```

Comment:
This implemented a fallback to the pure-python version if the cython version was missing.

I don't see this fallback in clru. I think you forgot to move it, clru should be doing this now, so importing "cuckoocache" would import the best version (cython if available, pure python if not)

Reply:
@klaussfreire ping about this? I changed the __init__ on cLRU